hera_mc/cm_hookup.py

```diff
@@ -202,22 +197,21 @@ class Hookup:
     hookup_list_to_cache and if the cache file is current relative to the cm_version
     """
     hookup_list_to_cache = cm_utils.all_hera_zone_prefixes
-    if six.PY2:
+    if six.PY2:  # pragma: no cover
```

Comment:
It does python 3 second, so it doesn't give you "credit" for that statement (i.e. coveralls final report shows it as uncovered).  I'm striving for 100%!

Reply:
coveralls should be combining the two reports. I'll see if I can figure out what's going on.

Reply:
It is combining them. If you look at the files, it shows a `2x` on most lines because they were hit in python 2 & 3, and only a `1x` on the python 3 line because it's only hit once. Can you take out this comment and so we can see what's going on? If it's not being hit by the python 2 tests we should figure out why not.